src/gamecontroller.cs

```diff
@@ -202,7 +226,7 @@ partial interface GCGamepadSnapshot {
 
 	[iOS (7,0)]
 	[Mac (10,9)]
-	[BaseType (typeof (NSObject))]
+	[BaseType (typeof (GCPhysicalInputProfile))]
```

Comment:
All this base changes are ok, but do remember that changing a base COULD be a breaking change. Not in this case since GCPhysicalInputProfile is a NSObject and it was the parent of the old class.

Reply:
It will be listed as a breaking change but there is nothing we can do about it, if a base changes we have to honor that

Reply:
This case technically should not be an issue. Others might: https://docs.microsoft.com/en-us/dotnet/core/compatibility/#types

>  REQUIRES JUDGMENT: Introducing a new base class
>
>A type can be introduced into a hierarchy between two existing types if it doesn't introduce any new abstract members or change the semantics or behavior of existing types. For example, in .NET Framework 2.0, the DbConnection class became a new base class for SqlConnection, which had previously derived directly from Component.